apis/control/v1alpha1/cassandratask_types.go

```diff
@@ -60,8 +60,13 @@ type CassandraTaskSpec struct {
 type CassandraCommand string
 
 const (
-	CommandCleanup CassandraCommand = "cleanup"
-	CommandRebuild CassandraCommand = "rebuild"
+	CommandCleanup         CassandraCommand = "cleanup"
```

Comment:
reconcile_racks.go only makes cleanup tasks in any case at the moment. But sure, that's not a big thing to fix (it doesn't necessarily work with other commands though, since it doesn't accept args for example).